pkg/models/app.go

```diff
@@ -3,11 +3,11 @@ package models
 // App is the model struct for apps
 // swagger:model App
 type App struct {
-	ID                    string    `json:"id"`
-	AppID                 string    `json:"appId"`
-	AppName               string    `json:"appName"`
-	NumOfDeployedVersions int64     `json:"numOfDeployedVersions,omitempty"`
-	NumOfClients          int64     `json:"numOfClients,omitempty"`
-	NumOfAppLaunches      int64     `json:"numOfAppLaunches,omitempty"`
-	DeployedVersions      []Version `json:"deployedVersions,omitempty"`
+	ID                    string     `json:"id,omitempty"`
+	AppID                 string     `json:"appId,omitempty"`
+	AppName               string     `json:"appName,omitempty"`
+	NumOfDeployedVersions int        `json:"numOfDeployedVersions"`
```

Comment:
@craicoverflow Hwo come the `omitempty` has been changed from here to the ID, AppID and AppName fields?

Reply:
@davidffrench Hmm I am not sure why it got added to the ID, AppID and AppName fields - possibly because this PR has merged a lot of changes from other branches in the past week or so and it got overwritten. They should not have `omitempty` though so I will remove.

Reply:
Thanks @craicoverflow . What was the reason for removing them from `NumOfDeployedVersions`, `NumOfClients` and `NumOfAppLaunches`?

Reply:
Because if their values are 0 then the values will not be sent back with in the payload. We want those values even if they will be displayed on the client UI.

Reply:
This may be a problem as we did not want them sent back on the `/apps/{id}` response which will be using the same model. It is not a big deal though if they are sent back in that request with no values.

Reply:
Pointers fixes this problem, updated PR.

https://stackoverflow.com/a/38487668

Reply:
That is really good to know, thank you